### DISCUSSION TITLE
Why yarn 2 is so different, it cant support most of my projects anymore?

### DISCUSSION THREAD

**Discussion by JenuelDev** [yarnpkg/berry#3019] [Q&A] [answered]
Hi, I am a web developer, and I have used npm and yarn 1 package managers. They work really fine, but when using yarn 2 it's really different you know, there is no more node_modules folder.  So I went back to yarn 1 or even using npm instead...

**Comment by devoto13** [comment, marked as answer]
Please check https://yarnpkg.com/getting-started/migration for the description of Plug'n'Play pros and cons and how to opt-in to use `node-modules` linker, which will install your dependencies using good ol' `node_modules` folder structure.

**Comment by kopax-polyconseil** [comment]
Just went to yarn 1 to. Lost 3h trying to upgrade in a 1 month old project. Too much to relearn. :/

**Comment by magnus-trent** [comment]
I just throw it into `nodeLinker: node-modules` and everything works just fine. I still have the cache, as well as the butter smooth terminal. Anything I feel like PnP supports I may use, but I mostly develop odd solutions for production software that definitely favor the `node_modules` setup.
